static/src/javascripts/lib/proximity-loader.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+
+import bonzo from 'bonzo';
+import debounce from 'lodash/functions/debounce';
+import fastdom from 'lib/fastdom-promise';
+import mediator from 'lib/mediator';
+
+let items = [];
+const scroll = { top: 0, bottom: 0 };
+
+const doProximityLoading = (): void => {
+    scroll.top = window.pageYOffset;
+    scroll.bottom = scroll.top + bonzo.viewport().height;
```

Comment:
will this trigger a DOM read?

Reply:
Probably yes. Since there are no specs for the file, I'd like to postpone adding fastdom for later ...

Reply:
why wait for a spec, out of interest?

Reply:
Because everything related to jest in combination with DOM measurement and scrolling is super painful, if not impossible to test ... It would block me for the rest of the week and I think there are more important conversions to be done ...